javascript/javascript3/week3/shopping-cart/shopping.js

```diff
@@ -0,0 +1,128 @@
+class Product {
+    constructor(name, price) {
+      this.name = name;
+      this.price = price;
+  }
+  convertToCurrency(currency) {
+    const p = document.createElement('h3');
+    const div = document.createElement('div');
+    div.appendChild(p);
+    document.body.appendChild(div);
+    if (currency.toLowerCase().includes('dollar')) {
+      p.innerHTML = `${this.name} : ${this.price * 0.16}$`;
+    } else if (currency.toLowerCase().includes('euro')) {
+      p.innerHTML = `${this.name} : ${this.price * 0.13}€`
+    } else if (currency.toLowerCase().includes('pound')) {
+      p.innerHTML = `${this.name} : ${this.price * 0.12}£`
+    } else {
+      p.innerHTML = this.price
+    }
+  }
+  }
+ 
+  class ShoppingCart {
+    constructor() {
+      this.products = [];
+    }
+  
+    addProduct(product) {
+      // Implement functionality here
+        this.products.push(product);
+    
+    }
+  
+    removeProduct(product) {
+      // Implement functionality here
+        const updatedProducts = this.products.filter((item) =>
+            item.name !== product.name)
+        this.products = updatedProducts;
+    }
+  
+      searchProduct(productName) {
+          //Implement functionality here
+         ;
+        const search = document.createElement('h3')        
+        const searchResult = this.products.filter((item) => {
+          if (item.name === productName) {
+            search.innerHTML = ` Search result : ${item.name}, Price : ${item.price}`;
+          }
+        })        
+          document.body.appendChild(search);
+       }
+  
+    getTotal() {
+      //Implement functionality here
+        let total = 0;
+        this.products.forEach((item) => total = item.price + total)
+        const totalPrice = document.createElement('p');
```

Comment:
You should return total here and move everything about rendering in renderProducts method.

Reply:
Resolved.